src/Kernel32.Desktop/Kernel32Extensions.cs

```diff
@@ -19,11 +20,16 @@ public static string GetMessage(this NTSTATUS status)
         {
             using (var ntdll = LoadLibrary("ntdll.dll"))
             {
+                int dwLanguageId = 0;
+#if DESKTOP
+                dwLanguageId = CultureInfo.CurrentCulture.LCID;
```

Comment:
I like the idea of treating the Unspecified value and other special values differently and solve it gracefully. The behavior I expect when I call GetMessage from these native errors is that they respect the culture I am already using on the managed code, so that the user can expect the same language on both managed and native errors.